ofrak_core/test_ofrak/components/test_free_space.py

```diff
@@ -19,11 +19,11 @@
 async def resource_under_test(ofrak_context: OFRAKContext) -> Resource:
     resource = await ofrak_context.create_root_resource(
         "mock_memory_region",
-        b"\xff" * 0x100,
+        b"\xff" * 0x110,
         (Program,),
     )
     memory_region = await resource.create_child_from_view(
-        MemoryRegion(0, 0x100), data_range=Range(0, 0x100)
+        MemoryRegion(0x10, 0x100), data_range=Range(0x10, 0x110)
```

Comment:
Does the memory region need to be 0x100 bytes long?

This is a small nit, but I think this bugfix would be easier to understand if only the start offset changed, and not the end offset with it.